pylatex/utils.py

```diff
@@ -249,54 +236,304 @@ def bold(s, *, escape=True):
     return NoEscape(r'\textbf{' + s + '}')
 
 
+def text_color(s, color, *, escape=True):
+    r"""Change the string color.
+
+    Args
+    ----
+    s : str
+        The string to be formatted
+    escape : bool
+        If true the text will be escaped
+    color : str
+        The name of the color for the text
+    """
+
+    if escape:
+        s = escape_latex(s)
+
+    return NoEscape(r"\textcolor{" + color + "}{" + s + "}")
+
+
 def italic(s, *, escape=True):
     r"""Make a string appear italicized in LaTeX formatting.
 
     italic() wraps a given string in the LaTeX command \textit{}.
-
     Args
     ----
     s : str
         The string to be formatted.
     escape: bool
         If true the italic text will be escaped
-
     Returns
     -------
     NoEscape
         The formatted string.
-
     Examples
     --------
     >>> italic("hello")
     '\\textit{hello}'
     >>> print(italic("hello"))
     \textit{hello}
-
     """
     if escape:
         s = escape_latex(s)
 
     return NoEscape(r'\textit{' + s + '}')
 
 
+def page_break():
+    r"""Add a page break to the current environment."""
+
+    return NoEscape(r'\newpage')
+
+
+def line_break():
+    r"""Add a line break to the current line."""
+
+    return NoEscape('\linebreak ')
+
+
+def new_line():
+    r"""Add a new line."""
+
+    return NoEscape(r'\newline ')
+
+
+def horizontal_fill():
+    r"""Fill the current line."""
+
+    return NoEscape(r'\hfill')
+
+
+def horizontal_skip(size):
+    r"""Add/remove the amount of horizontal space between elements.
+
+    Args
+    ----
+    size: str
+        The amount of horizontal space to add
+    """
+
+    return pylatex.base_classes.UnsafeCommand("hspace*", arguments=size)
+
+
+def display_page_number():
+    r"""Provide the page number in the following format: Page # of ##."""
+
+    return NoEscape(r'Page \thepage\ of \pageref{LastPage}')
+
+
+def huge(s, *, escape=True):
```

Comment:
Useful again, but add this to the "pylatex.basic" submodule as well and subclass the `Environment` class.